docs/source/datasources/usgs_ndvi.rst

```diff
@@ -0,0 +1,131 @@
+for cloud cover. Files for a specific dekad and region can range from 30MB up to over 100MB,
+so downloading and processing can take a long time.
+
+
+USGS publishes its data through its `web portal <https://earlywarning.usgs.gov/fews/datadownloads/East%20Africa/eMODIS%20NDVI%20C6>`_,
+which allows downloading data for a single geographical area and dekad. This data
+is extracted from the `back end data explorer <https://edcintl.cr.usgs.gov/downloads/sciweb1/shared/fews/web/africa/east/dekadal/emodis/ndvi_c6/temporallysmoothedndvi/downloads/monthly/>`_.
+This module is designed to allow programmatic access to and analysis of NDVI data from the data explorer.
+
+Usage
+-----
+
+To use this class, you first need to create a country configuration
+for the country you would like to analyze. You also need to supply the area
+of coverage that the country data is contained in. You can Make sure FEWS NET covers the country
+of interest. You can find area of coverage for NDVI by searching for eMODIS NDVI C6 on
+the  `USGS search portal <https://earlywarning.usgs.gov/fews/search>`_.
+The valid values of the area names are listed above under Background.
+
+An example country config for Ethiopia is:
+
+.. code-block:: python
+
+    iso3: eth
+    usgs_ndvi:
+        region_name: east-africa
+
+For a number of countries this config has been implemented already.
+For others, you can create a custom country config.
+Once you have created the config you can load it:
+
+.. code-block:: python
+
+    from aatoolbox import create_country_config
+    country_config = create_country_config(iso3="eth")
+
+Next you need to instantiate the USGS NDVI class with the country config. For this
+example, we will use the class for accessing temporally smoothed NDVI values. Since
+the size of the data we will be working with is so large, we will only look at a
+sample of data from a few dekads, corresponding to the dates in January:
+
+.. code-block:: python
+
+    from aatoolbox import UsgsNdviSmoothed
+    ndvi_smooth = UsgsNdviSmoothed(
+        country_config=country_config,
+        start_date="2021-01-01",
+        end_date="2021-01-31"
+    )
+
+We will first need to download the NDVI data if you haven't used the class before.
+The code downloads all data between `start_date` and `end_date`. Be careful if not
+setting start and end date during instantiation, the method will automatically
+look for data between the first available data in the 19th dekad of 2002 and today.
+We simply have to call the method to download available data.
+
+.. code-block:: python
+
+    ndvi_smooth.download()
+
+Since each NDVI file is downloaded as a .tif, we can load them in individually as
+rasters. We can do this for a specific date. This time we will specify the date
+as a year and dekad tuple, although specify as a `datetime.date` or string as
+above is equally appropriate.
+
+.. code-block:: python
+
+    ndvi_2021_01 = ndvi_smooth.load_raster(date=(2021, 01))
```

Comment:
is this trailing 0 necessary? If so, maybe dont leave that to the user but add that in the code (can use `:02d`)

Reply:
Ah no it isn't.